editoast/src/main.rs

```diff
@@ -150,6 +206,8 @@ async fn runserver(
     let is_sentry_initialized = _guard.is_some();
 
     let server = HttpServer::new(move || {
+        let auth = HttpAuthentication::bearer(validator);
+        let use_auth = env::var("OSRD_BACKEND_ENVIRONMENT").map_or(false, |v| v == "sncf_reseau");
```

Comment:
Instead of having another env var. We can simply check if the `OIDC_JWK_URIS` is given.